client/src/pages/Checkout.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
```

Comment:
This looks like it's from the other PR, probably want to make sure you branch from dev as well when starting a new feature!